include/common.h

```diff
@@ -92,9 +92,11 @@ typedef unsigned __int64	uint64_t;
 #include <string.h>
 #include <stdlib.h>
 #include <stdio.h>
+#include <stdint.h>
 #include <errno.h>
 #include <time.h>
 #include <ctype.h>
+#include <malloc.h>
```

Comment:
...ok, who the fuck uses malloc.h? #ifdef that shit - malloc() comes from stdlib.h unless you're stuck in last century.

Using malloc.h breaks BSD builds.